docs/project/project_edit.md

```diff
@@ -0,0 +1,80 @@
+# Project Editing
+
+Start by [navigating](project_introduction.md#project-navigation) to the project summary view of your project of interest.
+
+!!! info "project access"
+    Should you not see your project of interest, 
+    please make sure that you have been granted access to it by the project owner
+
+Within the project summary view, press the edit button on the top right to open up the edit dialog for your project.
+
+!!! info "project role"
+Should you not see the edit button,
```

Comment:
this part is not inside the "project role" box

Reply:
Dang spacing is important 🤦